components/Conversations/Panel/ConversationEntry.tsx

```diff
@@ -74,7 +75,7 @@ export function ConversationEntry({
         <h3 className="text-[15px] font-medium text-primary">{displayTitle}</h3>
       </div>
       {!isMergeActive && (
-        <div className="align-center hidden justify-center gap-[22px] group-hover:flex">
+        <div className="align-center hidden justify-center gap-[22px] text-tertiary group-hover:flex">
```

Comment:
share is commented out here but we have it working in another place?

Reply:
Yeah styling is all messed up here so disabled yesterday - working on that now is diff PR